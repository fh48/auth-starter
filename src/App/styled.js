import styled from 'styled-components';

export const Span = styled.span`
  font-family: sans-serif;
  font-size: 28px;
`;
