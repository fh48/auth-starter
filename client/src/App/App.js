import React, {Component, Fragment} from 'react';

import Login from './Login/';
import Signup from './Signup/';

import * as Styled from './styled';

class App extends Component {
  render() {
    return (
      <Fragment>
        <h1>Hello World!</h1>
        <Login />
        <Signup />
      </Fragment>
    );
  }
}

export default App;
