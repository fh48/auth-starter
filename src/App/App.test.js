import React from 'react';
import {shallow} from 'enzyme';

import App from './App';

describe('App', () => {
  let node;

  it('should render the application', () => {
    node = shallow(<App />);

    expect(node.debug()).toMatchSnapshot();
  });
});
