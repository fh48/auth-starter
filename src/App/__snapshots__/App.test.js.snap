// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the application 1`] = `
"<Symbol(react.fragment)>
  <h1>
    Hello World!
  </h1>
  <Login />
  <Signup />
</Symbol(react.fragment)>"
`;
